File: ChickenScratchEngine/Models/GameState.cs
using System;
using System.Collections.Generic;

namespace ChickenScratchEngine.Models
{
    public class GameState
    {
        public List<GamePlayer> Players { get; set; }
        public GamePlayer ActivePlayer { get; set; }
        public string CurrentWord { get; set; }
        public bool StartOfNewRound { get; set; }
        public DateTime TimeOfRoundEnd { get; set; }
    }
}

File: ChickenScratch/ClientApp2/src/app/models/gameState.ts
import { Player } from "./player";

export interface GameState {
    players: GamePlayer[],
    activePlayer: GamePlayer,
    currentWord: string,
    startOfNewRound: boolean,
    timeOfRoundEnd: Date
}

export interface GamePlayer {
    id: string;
    name: string;
    score: number;
    hasGuessedCorrectly: boolean;
}
